ui/raidboss/data/07-dt/eureka/occult_crescent_south_horn.txt

```diff
@@ -1037,7 +1037,7 @@ hideall "--sync--"
 15262.4 label "oc-extreme-prejudice-loop"
 15262.4 "Assail" Ability { id: "A1C9", source: "Command Urn" }
 15269.3 "Rockslide/Stone Swell?" #Ability { id: ["9A2F", "9A2E"], source: "Vassal Vessel" }
-15269.3 "--sync--" Ability { id: "9A2F", source: "Vassal Vessel" } jump 15269.3
+15269.3 "--sync--" Ability { id: "9A2F", source: "Vassal Vessel" } jump 15369.3
```

Comment:
`jump` also supports jumping to a label instead of a fixed time like `forcejump`, which would help prevent time typos like this.